src/purchases.ts

```diff
@@ -105,9 +105,9 @@ export enum BILLING_FEATURE {
  */
 export interface LogInResult {
   /**
-   * The Purchaser Info for the user.
```

Comment:
addAttributionData should be removed from this file too, right?

Reply:
Good catch.